webpack.config.js

```diff
@@ -1,13 +1,16 @@
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const path = require("path");
+
+const srcDir = path.resolve(__dirname, "src");
 
 module.exports = (env, options) => {
     const mode = options.mode || 'development'
-    const masterURL = mode == 'production' ? 'https://hslo.sigr.io/' : 'http://127.0.0.1:5500/dist/'
+    const masterURL = mode == 'production' ? '/' : './' // you can just open that normally 

```

Comment:
absolute path needed because hslo site and agar.io site is different

Reply:
ok